CHANGELOG.md

```diff
@@ -1,14 +1,21 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [0.3.1] - 2023-07-25
+
+### Changed
+
+ - Property application.security.username deleted, instead the fixed value idsUser is used
+ 
```

Comment:
What about hardcoding to TLS only mode?
And also in readme, that it is secure endpoints only